Frameworks/QuartzCore/CATransaction.mm

```diff
@@ -36,14 +36,15 @@
 
```

Comment:
Nits about these guys:

* `__declspec(thread)` objective-C things will not be properly ref-counted.
* they're being compared against `NULL` (not even `nullptr`!) instead of `nil` everywhere in this file.